src/rules/valid-expect-in-promise.ts

```diff
@@ -187,14 +182,13 @@ export default createRule<unknown[], MessageIds>({
           }
 
           const testFunctionBody = body.body;
-          const [fulfillmentCallback, rejectionCallback] = node.arguments;
```

Comment:
should this be a fix (i.e. released as a patch?)?

Reply:
Na, this is just a `refactor` as it doesn't actually change the behaviour: since we pick these two items out of the array only to then make a new array out of them which we pass through to a function.

So `.slice(0, 2)` should be functionally equivalent as that selects the first two items in the `node.arguments` array.